test/source/tests/compose.ts

```diff
@@ -2545,6 +2548,16 @@ export const defineComposeTests = (testVariant: TestVariant, testWithBrowser: Te
     test(
       'user3@standardsubdomainfes.localhost:8001 - PWD encrypted message with FES web portal - pubkey recipient in bcc',
       testWithBrowser(undefined, async (t, browser) => {
+        t.mockApi?.setHandlers({
+          ...mockGoogleEndpoints,
+          // ...mockBackendEndpoints,
+          ...mockAttesterEndpoints,
+          ...mockKeyManagerEndpoints,
+          // ...mockWkdEndpoints,
+          // ...mockSksEndpoints,
+          ...mockCustomerUrlFesEndpoints,
+          // ...mockSharedTenantFesEndpoints,
+        });
```

Comment:
I didn't mean this way, setting handlers.

I meant we would be setting configuration. Since now we have one MockApi per per test, we have an opportunity to configure mock api with different data (or expectations).

Like this:

```ts
    mockApi.fesConfig = { ... }; // particular client configuration for this test
    mockApi.ekmConfig = { returnKeys: [ekmKeySamples.e2eValidKey.prv] } // same
    mockApi.attesterConfig = { ... }; // configure Attester with particular public keys for this test 
    mockApi.googleConfig = {
      accounts: {
        'e2e.enterprise.test@flowcrypt.com': { // which account is allowed for Gmail login in this test
          messages: ['CC and BCC test'], // exported messages pre-loaded for this test, by subject
        }
      }
    }
```

notice it's receiving information on what to return (per service), not receiving the endpoints directly as done in the PR.

Have a look at any iOS mock tests, like these:
https://github.com/FlowCrypt/flowcrypt-ios/blob/master/appium/tests/specs/mock/composeEmail/CheckInvalidEmailRecipient.spec.ts
https://github.com/FlowCrypt/flowcrypt-ios/blob/master/appium/tests/specs/mock/composeEmail/CheckSendAsAlias.spec.ts

Notice how each test has some unique mock api configuration to work with, so it's clear which data the test needs.

Reply:
@sosnovsky

Reply:
Yes, currently it's not exposed, I was planning to add it to `AvaContext`:
```ts
t.mockApi = await startMockApiAndCopyBuild(t);
```
(here - https://github.com/FlowCrypt/flowcrypt-browser/blob/master/test/source/test.ts#L68)

And then use it in tests like this:
```ts
t.mockApi.fesConfig = { ... };
t.mockApi.ekmConfig = { ... };
```